lib/core/versions/latest/UpdateOperation.ts

```diff
@@ -2,6 +2,8 @@ import DeltaModel from './models/DeltaModel';
 import Encoder from './Encoder';
 import ErrorCode from './ErrorCode';
 import JsonAsync from './util/JsonAsync';
+import Jwk from './util/Jwk';
+import JwkEs256k from '../../models/JwkEs256k';
 import Jws from './util/Jws';
 import Multihash from './Multihash';
 import Operation from './Operation';
```

Comment:
Search for `const expectKidInHeader = true;` in this file, it needs to be `false` now to be consistent with other operations. Once this is done, you'll probably need to update `Jws` class and related method to ensure it's code coverage remains at 100%.